plugin-settings/README.md

```diff
@@ -12,10 +12,30 @@ plugins {
 
 ### Project discovery
 
-The plugin will automatically discover all projects by finding their build files and include them. This means it is 
-not necessary anymore to manually add `include("...")` for each project to `settings.gradle`. The requirement for this
+The plugin will automatically discover all projects by finding their build files and include them. This means it is
+not necessary anymore to manually add `include("...")` for each project to `settings.gradle.kts`. The requirement for this
 to work is that the build file is named after the path. For example the project `:foo:bar` should have a build file
-called `foo-bar.gradle` (or `foo-bar.gradle.kts`).
+called `foo-bar.gradle.kts` (or `foo-bar.gradle`). `.gradle.kts` discoverability is enabled by default and `.gradle`
+discoverability can be enabled with:
+
+```groovy
+freeletics {
+    discoverProjects(kts = false)
+}
+```
+
+Mixing `.gradle.kts` and `.gradle` files for automatic discoverability is not supported.
+
+Instead of using project root directory as a base for automatic discoverability it is possible to only include
+certain subdirectories with:
+
+```kotlin
+freeletics {
+    it.discoverProjectsIn(
+        directories = arrayOf("app", "features")
+    )
+}
+```
```

Comment:
For this you also need to set this property to disable the default discovery `fgp.discoverProjects.automatically=false`

Reply:
added this 👍